modules/database/migrations.py

```
@@ -0,0 +1,37 @@
+    "5.8.0": initial_setup,
+}
+
+
+def run_migrations(database_path: str) -> bool:
+    database = db.DatabaseRepository(database_path=database_path)
+
+    current_version = "0.0.0"
+    try:
+        current_version = database.get_database_version()
+    except Exception as e:
+        pass
```

Ruff (0.8.0)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)